imports/plugins/core/payments/server/no-meteor/mutations/__snapshots__/approveOrderPayments.test.js.snap

```diff
@@ -0,0 +1,13 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`throws if orderId isn't supplied 1`] = `"Order ID is required"`;
+
+exports[`throws if paymentIds isn't supplied 1`] = `"Payment ids is required"`;
+
+exports[`throws if permission check fails 1`] = `"Access denied"`;
+
+exports[`throws if shopId isn't supplied 1`] = `"Shop ID is required"`;
+
+exports[`throws if the order doesn't exist 1`] = `"Access denied"`;
```

Comment:
This isn't the expected error. You might need to mock more things to be sure it gets past the permission check and hits the order exists check.

Reply:
Updated